### DISCUSSION TITLE
modify the used storage bank (9) and MIDI-channel (10)

### DISCUSSION THREAD

**Discussion by Zedrock** [raphaelquast/beatstep#47] [Q&A] [answered]
Hello,
Got a question in mind : May I use an other storage bank for BeatstepQ settings : the 16 one instead of the 9 ? 
Is there any hardware reason to that use of bank 9 ?
Or this is because it is the drum MIDI channel (10) ?

Zed'

**Comment by raphaelquast** [comment, marked as answer]
Hey,
no there is no particular reason for using storage-bank 9. You can change it to any of the 16 slots available.
(The same is true for the MIDI-channel... it can be changed to any channel you like.)
I simply tried to pick a channel and a slot that is not used by most users.

To change the used memory-slot, simply change line `24` of  `BeatStep_Q.py` to the slot-number you want:
NOTICE: python starts indexing at 0 so the number 8 corresponds to memory-slot 9 ! (same for the MIDI channel)
```python
21    # the midi-channel to use for buttons and encoders
22    CHANNEL = 9
23    # the used memory-slot to store the configurations
24    MEMORY_SLOT = 8
```

Cheers, Raphael
